### ISSUE TITLE
[Planned Feature] Button to upload logs to a server.

### ISSUE THREAD

**Issue by cbartondock** [doZennn/steam-rom-manager#298]
@doZennn let me know how you want to do this, I can pretty quickly add an upload logs button once the server is in place to receive them.

**Comment by doZennn** [comment]
How much separate files are you planning to upload? Is it just going to be a single text file?

**Comment by cbartondock** [comment]
I was thinking the logs, the OS, and the user configs. I can put them all in a single json if that would be easiest:
```typescript
{
  logText: string,
  userConfigs: UserConfig[],
  systemInfo: { os: string, installType: string}
}
```

**Comment by cbartondock** [comment]
Implemented in 2.3.38.
